README.md

```diff
@@ -11,10 +11,31 @@ for help getting started.
 Visit the [Servo Project page](https://servo.org/) for news and guides.
 
 ## Getting Servo
+The first step is to `fork the repository` to your GitHub account. After forking, proceed to clone the repository using the instructions provided below.
+
 ``` sh
-git clone https://github.com/servo/servo
+git clone https://github.com/<your username >/servo.git
+```
+
+````{note}
+To address potential barriers with slow internet connections due to the large size of this repository, you can use [partial clones](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/#). Specifically, focusing on blobless clones, which are efficient for developers, involves utilizing the `--filter=blob:none` option in the git clone command.
+
+By using `--filter=blob:none`, the initial git clone operation downloads all reachable commits and trees, while blobs (file contents) for commits are only downloaded when performing a git checkout
+
+Here's the command to use blobless clones:
+
+```console
+git clone --filter=blob:none https://github.com/<your username>/servo.git
+
+```
+````
+
+> Navigate into the cloned repository folder 
+
+```console
 cd servo
 ```
```

Comment:
This is a pretty simple thing. Maybe remove the section changing directories and before the list of platforms below we can have a sentence that says:

"Ensure that you are in the Servo directory you have cloned and run the following steps for your platform:"

Reply:
Just addressed the changes 
Thank you